tests/global_config.py

```diff
@@ -7,6 +7,7 @@
 aws_compute_machine_type = "m5.xlarge"
 rosa_number_of_nodes = 2
 cloud_provider = "aws"
+kubeadmin_token = None
```

Comment:
Add comment that this is for acm_observability tests

Reply:
done